### ISSUE TITLE
Timeout while running calabash on jenkins and via SSH

### ISSUE THREAD

**Issue by Opalo** [calabash/calabash-ios#251]
Hi guys,
I have very similar problem to the already described in: https://github.com/calabash/calabash-ios/issues/203.

What's going on:
- running 'cucumber' locally works
- when I connect to the server using remote desktop and run 'cucumber' everything works fine
- when I connect to remote server via SSH and run 'cucumber' it fails with timeout
- when I run a job on jenkins (master - slave) CI server it also fails with timeout - there's a user logged in all the time to keep the 'visual' connection

How it fails exactly (via SSH connection and with jenkins):

What I've already tried:
- running it with sudo - no success
- running 'sudo DevToolsSecurity -enable'  - no success

How it fails:
When I start calabash console and run the server I get the following message (quite strange, to be honest)

```
Calabash::Cucumber::Launcher::StartError: Calabash::Cucumber::Launcher::StartError: Time out waiting for UIAutomation run-loop to Start. 
 Logfile /var/folders/0n/h1r75_2n2ynfc8q4y4d53y1h0000gq/T/run_loop20131122-13530-15kzx5g/run_loop.out 

 Failed to authorize rights (0x2) with status: -60007.
Failed to authorize rights (0x2) with status: -60007.
2013-11-22 11:59:53.566 ScriptAgent[13751:2e07] CLTilesManagerClient: initialize, sSharedTilesManagerClient
2013-11-22 11:59:53.566 ScriptAgent[13751:2e07] CLTilesManagerClient: init
2013-11-22 11:59:53.566 ScriptAgent[13751:2e07] CLTilesManagerClient: reconnecting, 0x9629af0
2013-11-22 11:59:53.636 ScriptAgent[13751:2e07] AXError: Could not auto-register for pid status change
2013-11-22 11:00:04 +0000 Fail: Could not start script, target application is not frontmost.
```

What makes me uneasy are those <b>'authorize'</b> lines as well as <b>'Fail: Could not start script, target application is not frontmost'</b>

Regardless how I try to run calabash tests simulator on the remote server is always started - so this isn't probably and issue with permissions.

Any ideas how it could be fixed?

**Comment by biolinh** [comment]
Hi @gsaslis. thank you for your answer.

I did as you suggest,  I bypassed the error. However, It shows another error as below 

xcodebuild[12244:1007] [MT] DVTAssertions: ASSERTION FAILURE in /SourceCache/DVTiOSFrameworks/DVTiOSFrameworks-6760/DVTiPhoneSimulatorRemoteClient/DTiPhoneSimulatorSession.m:285
Details:  iPhoneSimulatorSession logic error: listening for session started notification when sessionLifecycleProgress was 3
Object:   <DTiPhoneSimulatorSession: 0x7fa68f2ed4b0>
Method:   -_startSessionInSimulatorWithError:
Thread:   <NSThread: 0x7fa688c14b40>{name = (null), num = 1}
Hints: None
